### PR TITLE
Fix line lengths for comments and docstrings

### PR DESCRIPTION
**Pull request by jleonatti**
This is the last PR for refactoring line lengths. Now the flake8 directive E501 can be applied.

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [flake8]
-# To be added once code is refactored: E501
+# To be added once code is refactored:
```
Comment (akaszynski) [inline .flake8:2]: Still need this line?

File: ansys/dpf/core/core.py [hunk 2]
```diff
@@ -247,7 +247,8 @@ def _description(dpf_entity_message, server=None):
 
     Parameters
     ----------
-    dpf_entity_message : core.Operator._message, core.Workflow._message, core.Scoping._message, core.Field._message,
+    dpf_entity_message : core.Operator._message, core.Workflow._message,
+    core.Scoping._message, core.Field._message,
     core.FieldContainer._message, core.MeshedRegion._message...
```
Comment (akaszynski) [inline ansys/dpf/core/core.py:252]:
```suggestion
    dpf_entity_message : core.Operator._message, core.Workflow._message,
                         core.Scoping._message, core.Field._message,
                         core.FieldContainer._message, core.MeshedRegion._message
```

File: ansys/dpf/core/core.py [hunk 3]
```diff
@@ -419,7 +421,8 @@ def _description(self, dpf_entity_message):
 
         Parameters
         ----------
-        dpf_entity_message : core.Operator._message, core.Workflow._message, core.Scoping._message, core.Field._message,
+        dpf_entity_message : core.Operator._message, core.Workflow._message,
+        core.Scoping._message, core.Field._message,
```
Comment (akaszynski) [inline ansys/dpf/core/core.py:425]:
```suggestion
        dpf_entity_message : core.Operator._message, core.Workflow._message,
                             core.Scoping._message, core.Field._message,
```

File: ansys/dpf/core/dpf_operator.py [hunk 4]
```diff
@@ -111,7 +111,9 @@ def connect(self, pin, inpt, pin_out=0):
         ----------
         pin : int
             Number of the input pin.
-        inpt :  str, int, double, bool, list of int, list of doubles, Field, FieldsContainer, Scoping, ScopingsContainer, MeshedRegion, MeshesContainer, DataSources, Operator
+        inpt :  str, int, double, bool, list of int, list of doubles,
+        Field, FieldsContainer, Scoping, ScopingsContainer, MeshedRegion,
+        MeshesContainer, DataSources, Operator
```
Comment (akaszynski) [inline ansys/dpf/core/dpf_operator.py:116]:
```suggestion
        inpt : str, int, double, bool, list of int, list of doubles,
               Field, FieldsContainer, Scoping, ScopingsContainer, MeshedRegion,
               MeshesContainer, DataSources, Operator
```

File: ansys/dpf/core/examples/downloads.py [hunk 5]
```diff
@@ -348,15 +353,17 @@ def download_extrapolation_3d_result() -> dict:
...
+    for extrapolation of 2d-element and return the dictionary of 2 download paths.
 
     Examples files are downloaded to a persistent cache to avoid
     re-downloading the same file twice.
 
     Returns
     -------
     dict
-        containing path to the example file of ref and path to the example file of integrated points.
+        containing path to the example file of ref and path to the example
```
Comment (akaszynski) [inline ansys/dpf/core/examples/downloads.py:365]:
```suggestion
        Contains path to the example file of ref and path to the example
```

File: ansys/dpf/core/field_base.py [hunk 6]
```diff
@@ -492,7 +492,8 @@ def _set_data(self, data):
                     != data.size / self.component_count
                 ):
                     raise ValueError(
-                        f"An array of shape {self.shape} is expected and shape {data.shape} is in input"
+                        f"An array of shape {self.shape} is expected and "
+                        f"shape {data.shape} is in input"
```
Comment (akaszynski) [inline ansys/dpf/core/field_base.py:496]:
```suggestion
                        f"shape {data.shape} was input"
```

File: ansys/dpf/core/field_base.py [hunk 7]
```diff
@@ -587,7 +588,8 @@ def get_entity_data(self, index):
         """
         if index > self._num_entities:
             raise ValueError(
-                f"asked scoping {index} is greater than the number of available indices {len(self._scoping_ids_copy)}"
+                f"asked scoping {index} is greater than the number of "
```
Comment (akaszynski) [inline ansys/dpf/core/field_base.py:591]:
```suggestion
                f"Requested scoping {index} is greater than the number of "
```

File: ansys/dpf/core/field_base.py [hunk 8]
```diff
@@ -768,7 +772,8 @@ def data(self, data):
             if isinstance(data, (np.ndarray, np.generic)):
                 if data.shape != self.shape and 0 != self.size:
                     raise ValueError(
-                        f"An array of shape {self.shape} is expected and shape {data.shape} is in input"
+                        f"An array of shape {self.shape} is expected and "
+                        f"shape {data.shape} is in input"
```
Comment (akaszynski) [inline ansys/dpf/core/field_base.py:776]:
```suggestion
                        f"shape {data.shape} was input"
```

File: ansys/dpf/core/inputs.py [hunk 9]
```diff
@@ -53,7 +53,8 @@ def connect(self, inpt):
                 inpt = inpt.outputs
             else:
                 raise ValueError(
-                    "The operator to connect in input has no outputs available, it cannot be connected"
+                    "The operator to connect in input has no outputs "
+                    "available, it cannot be connected"
```
Comment (akaszynski) [inline ansys/dpf/core/inputs.py:57]:
```suggestion
                    "available. It cannot be connected."
```

File: ansys/dpf/core/inputs.py [hunk 10]
```diff
@@ -169,7 +173,9 @@ def connect(self, inpt):
 
         Parameters
         ----------
-        inpt : str, int, double, Field, FieldsContainer, Scoping, DataSources, MeshedRegion, ScopingsContainer, CyclicSupport, ..., Output, Outputs, Operator
+        inpt : str, int, double, Field, FieldsContainer, Scoping,
+        DataSources, MeshedRegion, ScopingsContainer, CyclicSupport,
+        ..., Output, Outputs, Operator
```
Comment (akaszynski) [inline ansys/dpf/core/inputs.py:178]:
```suggestion
        inpt : str, int, double, Field, FieldsContainer, Scoping,
               DataSources, MeshedRegion, ScopingsContainer, CyclicSupport,
               ..., Output, Outputs, Operator
```

File: ansys/dpf/core/inputs.py [hunk 11]
```diff
@@ -178,7 +184,8 @@ def connect(self, inpt):
                 inpt = inpt.outputs
             else:
                 raise ValueError(
-                    "The operator to connect in input has no outputs available, it cannot be connected"
+                    "The operator to connect in input has no outputs "
+                    "available, it cannot be connected"
```
Comment (akaszynski) [inline ansys/dpf/core/inputs.py:188]:
```suggestion
                    "available. It cannot be connected."
```

File: ansys/dpf/core/meshed_region.py [hunk 12]
```diff
@@ -246,7 +246,9 @@ def named_selection(self, named_selection):
                 return op.get_output(0, types.scoping)
             else:
                 raise Exception(
-                    "getting a named selection from a meshed region is only implemented for meshed region created from a model for server version 2.0. Please update your server."
+                    "getting a named selection from a meshed region is "
```
Comment (akaszynski) [inline ansys/dpf/core/meshed_region.py:249]:
```suggestion
                    "Getting a named selection from a meshed region is "
```

File: ansys/dpf/core/meshed_region.py [hunk 13]
```diff
@@ -302,7 +304,8 @@ def _as_vtk(self, as_linear=True, include_ids=False):
             from ansys.dpf.core.vtk_helper import dpf_mesh_to_vtk
         except ModuleNotFoundError:
             raise ModuleNotFoundError(
-                "to use plotting capabilities, please install pyvista with :\n pip install pyvista>=0.24.0"
+                "to use plotting capabilities, please install pyvista "
```
Comment (akaszynski) [inline ansys/dpf/core/meshed_region.py:307]:
```suggestion
                "To use plotting capabilities, please install pyvista "
```

File: ansys/dpf/core/meshes_container.py [hunk 14]
```diff
@@ -14,7 +14,8 @@ class MeshesContainer(Collection):
 
     Parameters
     ----------
-    meshes_container : ansys.grpc.dpf.collection_pb2.Collection or ansys.dpf.core.MeshesContainer, optional
+    meshes_container : ansys.grpc.dpf.collection_pb2.Collection or
+    ansys.dpf.core.MeshesContainer, optional
```
Comment (akaszynski) [inline ansys/dpf/core/meshes_container.py:18]:
```suggestion
    meshes_container : ansys.grpc.dpf.collection_pb2.Collection or
                       ansys.dpf.core.MeshesContainer, optional
```

File: ansys/dpf/core/plotter.py [hunk 15]
```diff
@@ -102,12 +102,14 @@ def plot_chart(self, fields_container):
             import matplotlib.pyplot as pyplot
         except ModuleNotFoundError:
             raise ModuleNotFoundError(
-                "to use plot_chart capabilities, please install matplotlib with :\n pip install matplotlib>=3.2"
+                "to use plot_chart capabilities, please install "
```
Comment (akaszynski) [inline ansys/dpf/core/plotter.py:105]:
```suggestion
                "To use plot_chart capabilities, please install "
```

File: ansys/dpf/core/plotter.py [hunk 16]
```diff
@@ -260,7 +262,8 @@ def plot_contour(
             import pyvista as pv
         except ModuleNotFoundError:
             raise ModuleNotFoundError(
-                "to use plotting capabilities, please install pyvista with :\n pip install pyvista>=0.24.0"
+                "to use plotting capabilities, please install pyvista "
```
Comment (akaszynski) [inline ansys/dpf/core/plotter.py:265]:
```suggestion
                "To use plotting capabilities, please install pyvista "
```

File: ansys/dpf/core/plotter.py [hunk 17]
```diff
@@ -291,7 +294,8 @@ def _plot_contour_using_vtk_file(self, fields_container, notebook=None):
...
```
Comment (akaszynski) [inline ansys/dpf/core/plotter.py:297]:
```suggestion
                "To use plotting capabilities, please install pyvista "
```

File: ansys/dpf/core/scopings_container.py [hunk 18]
```diff
@@ -15,8 +15,10 @@ class ScopingsContainer(Collection):
 
     Parameters
     ----------
-    scopings_container : ansys.grpc.dpf.collection_pb2.Collection or ansys.dpf.core.ScopingsContainer, optional
-        Create a scopings container from a Collection message or create a copy from an existing scopings container
+    scopings_container : ansys.grpc.dpf.collection_pb2.Collection or
+    ansys.dpf.core.ScopingsContainer, optional
```
Comment (akaszynski) [inline ansys/dpf/core/scopings_container.py:19]:
```suggestion
    scopings_container : ansys.grpc.dpf.collection_pb2.Collection or
                         ansys.dpf.core.ScopingsContainer, optional
```

File: ansys/dpf/core/workflow.py [hunk 19]
```diff
@@ -73,7 +73,8 @@ def connect(self, pin_name, inpt, pin_out=0):
         pin_name : str
             Name of the pin to connect. This name should be
             exposed before with wf.set_input_name
-        inpt : str, int, double, bool, list of int, list of doubles, Field, FieldsContainer, Scoping, ScopingsContainer,
+        inpt : str, int, double, bool, list of int, list of doubles,
+        Field, FieldsContainer, Scoping, ScopingsContainer,
```
Comment (akaszynski) [inline ansys/dpf/core/workflow.py:77]:
```suggestion
        inpt : str, int, double, bool, list of int, list of doubles,
               Field, FieldsContainer, Scoping, ScopingsContainer,
```

File: ansys/dpf/core/workflow.py [hunk 20]
```diff
@@ -240,7 +241,8 @@ def add_operators(self, operators):
             request.operators.extend([operators._message])
         else:
             raise TypeError(
-                f"operators to add to the workflow are expected to be of type {type(list).__name__} or {type(dpf_operator.Operator).__name__}"
+                "operators to add to the workflow are expected to be of "
```
Comment (akaszynski) [inline ansys/dpf/core/workflow.py:244]:
```suggestion
                "Operators to add to the workflow are expected to be of "
```

File: examples/03-advanced/04-extrapolation_stress_3d.py [hunk 21]
```diff
@@ -3,16 +3,25 @@
...
-Extrapolating results available at Gauss or quadrature points to nodal points for a field or fields container. The available elements are : linear quadrangle, parabolic quadrangle, linear hexagonal, quadratic hexagonal, linear tetrahedral, quadratic tetrahedral.
+This example shows how to compute the nodal components stress from
+Gaussian points (integration points) for 3D-element by using the method
+of extrapolation.
 
-1st step : get the data source's analyse of integration points(this file was add the commands APDL "EREXS, NO").
+Extrapolating results available at Gauss or quadrature points to nodal
+points for a field or fields container. The available elements are:
+linear quadrangle, parabolic quadrangle, linear hexagonal,
+quadratic hexagonal, linear tetrahedral, quadratic tetrahedral.
```
Comment (akaszynski) [inline examples/03-advanced/04-extrapolation_stress_3d.py:13]:
```suggestion
points for a field or fields container. The available elements are:

* Linear quadrangle
* Parabolic quadrangle
* Linear hexagonal
* Quadratic hexagonal
* Linear tetrahedral
* Quadratic tetrahedral.
```

File: examples/03-advanced/04-extrapolation_stress_3d.py [hunk 22]
```diff
@@ -3,16 +3,25 @@
...
+
+1st step : get the data source's analyse of integration points(this file
+was add the commands APDL "EREXS, NO").
```
Comment (akaszynski) [inline examples/03-advanced/04-extrapolation_stress_3d.py:16]:
```suggestion
1st step : Get the data source's solution from the integration points (this
result file was generated with the MAPDL option ``EREXS, NO``).
```

File: examples/03-advanced/04-extrapolation_stress_3d.py [hunk 23]
```diff
@@ -3,16 +3,25 @@
...
 
 2nd step: using operator of extrapolation to compute the nodal stress.
```
Comment (akaszynski) [inline examples/03-advanced/04-extrapolation_stress_3d.py:18]:
```suggestion
2nd step: Use operator of extrapolation to compute the nodal stress.
```

File: examples/03-advanced/04-extrapolation_stress_3d.py [hunk 24]
```diff
@@ -3,16 +3,25 @@
...
 
-3rd step: get nodal stress result from data source's analyse reference. The analyse was computed by ANSYS Mechanical APDL.
+3rd step: get nodal stress result from data source's analyse reference.
+The analyse was computed by ANSYS Mechanical APDL.
```
Comment (akaszynski) [inline examples/03-advanced/04-extrapolation_stress_3d.py:21]:
```suggestion
3rd step: Get nodal stress result from data source's analysis reference.
The analysis was computed by Ansys Mechanical APDL.
```

File: examples/03-advanced/04-extrapolation_stress_3d.py [hunk 25]
```diff
@@ -3,16 +3,25 @@
...
 
-4th step: compare the results between nodal stress from data source ref and nodal stress computed by Extrapolation Method.
+4th step: compare the results between nodal stress from data source ref
+and nodal stress computed by Extrapolation Method.
```
Comment (akaszynski) [inline examples/03-advanced/04-extrapolation_stress_3d.py:24]:
```suggestion
4th step: Compare the results between nodal stress from data source
reference and nodal stress computed by the extrapolation method.
```

File: examples/03-advanced/04-extrapolation_stress_3d.py [hunk 26]
```diff
@@ -45,7 +54,8 @@
 ###############################################################################
 # Extrapolation from integration points for stress result
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-# In this example we compute nodal component stress result from integration points's stress by using operator gauss_to_node_fc
+# In this example we compute nodal component stress result from
+# integration points's stress by using operator gauss_to_node_fc
```
Comment (akaszynski) [inline examples/03-advanced/04-extrapolation_stress_3d.py:58]:
```suggestion
# integration points stress by using the ``gauss_to_node_fc`` operator.
```

File: examples/03-advanced/04-extrapolation_stress_3d.py [hunk 27]
```diff
@@ -55,8 +65,13 @@
 ###############################################################################
 # Nodal stress result of integration points:
 ###############################################################################
-# The command "ERESX,NO" in Mechanical APDL is used to copy directly the gaussian (integration) points results to the nodes, instead of the results at nodes or elements (which are interpolation of results at a few gauss points).
-# The following plot shows the nodal values which are the averaged values of stresses at each node. The value shown at the node is the average of the stresses from the gaussian points of each element that it belongs to.
+# The command "ERESX,NO" in Mechanical APDL is used to copy directly the
```
Comment (akaszynski) [inline examples/03-advanced/04-extrapolation_stress_3d.py:68]:
```suggestion
# The command ``ERESX,NO`` in Mechanical APDL is used to copy directly the
```

File: examples/03-advanced/04-extrapolation_stress_3d.py [hunk 28]
```diff
@@ -65,7 +80,8 @@
 
 
 ###############################################################################
-# Create operator gauss_to_node_fc and compute nodal component stress by applying Extrapolation Method
+# Create operator gauss_to_node_fc and compute nodal component stress
```
Comment (akaszynski) [inline examples/03-advanced/04-extrapolation_stress_3d.py:83]:
```suggestion
# Create operator ``gauss_to_node_fc`` and compute nodal component stress
```

File: examples/03-advanced/04-extrapolation_stress_3d.py [hunk 29]
```diff
@@ -65,7 +80,8 @@
...
+# by applying Extrapolation Method
```
Comment (akaszynski) [inline examples/03-advanced/04-extrapolation_stress_3d.py:84]:
```suggestion
# by applying the extrapolation method.
```

File: examples/03-advanced/04-extrapolation_stress_3d.py [hunk 30]
```diff
@@ -105,7 +121,8 @@
 # Compare the stress result computed by extrapolation and reference's result.
 # Check if two fields container are identical.
 # Maximum tolerance gap between to compared values: 1e-2.
-# Smallest value which will be considered during the comparison step : all the abs(values) in field less than 1e-8 is considered as null
+# Smallest value which will be considered during the comparison
+# step : all the abs(values) in field less than 1e-8 is considered as null
```
Comment (akaszynski) [inline examples/03-advanced/04-extrapolation_stress_3d.py:125]:
```suggestion
# step : all the ``abs(values)`` in field less than 1e-8 is considered as null
```

File: examples/03-advanced/05-extrapolation_strain_2d.py [hunk 31]
```diff
@@ -3,16 +3,25 @@
 
 Extrapolation Method for strain result of 2D-element
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-This example shows how to compute the nodal components elastic strain from Gaussian points (integration points) for 2D-element by using the method of extrapolation.
-Extrapolating results available at Gauss or quadrature points to nodal points for a field or fields container. The available elements are : linear quadrangle, parabolic quadrangle, linear hexagonal, quadratic hexagonal, linear tetrahedral, quadratic tetrahedral.
+This example shows how to compute the nodal components elastic strain
```
Comment (akaszynski) [inline examples/03-advanced/05-extrapolation_strain_2d.py:6]:
```suggestion
This example shows how to compute the nodal component elastic strain
```

File: examples/03-advanced/05-extrapolation_strain_2d.py [hunk 32]
```diff
@@ -3,16 +3,25 @@
...
+from Gaussian points (integration points) for 2D-element by using the
+method of extrapolation.
 
-1st step : get the data source's analyse of integration points(this file was add the commands APDL "EREXS, NO").
+Extrapolating results available at Gauss or quadrature points to nodal
+points for a field or fields container. The available elements are:
+linear quadrangle, parabolic quadrangle, linear hexagonal,
+quadratic hexagonal, linear tetrahedral, quadratic tetrahedral.
```
Comment (akaszynski) [inline examples/03-advanced/05-extrapolation_strain_2d.py:13]:
```suggestion
points for a field or fields container. The available elements are:

* Linear quadrangle
* Parabolic quadrangle
* Linear hexagonal
* Quadratic hexagonal
* Linear tetrahedral
* Quadratic tetrahedral
```

File: examples/03-advanced/05-extrapolation_strain_2d.py [hunk 33]
```diff
@@ -3,16 +3,25 @@
...
+
+1st step : get the data source's analyse of integration points(this file
+was add the commands APDL "EREXS, NO").
```
Comment (akaszynski) [inline examples/03-advanced/05-extrapolation_strain_2d.py:16]:
```suggestion
1st step : Get the values at the data source's of integration points (this result
file was generated from MAPDL with ``EREXS, NO``).
```

File: examples/03-advanced/05-extrapolation_strain_2d.py [hunk 34]
```diff
@@ -3,16 +3,25 @@
...
 
 2nd step: using operator of extrapolation to compute the nodal elastic strain.
 
-3rd step: get nodal elastic strain result from data source's analyse reference. The analyse was computed by ANSYS Mechanical APDL.
+3rd step: get nodal elastic strain result from data source's analyse reference.
+The analyse was computed by ANSYS Mechanical APDL.
```
Comment (akaszynski) [inline examples/03-advanced/05-extrapolation_strain_2d.py:21]:
```suggestion
3rd step: Get the nodal elastic strain result from the data source.
The analysis was computed by Ansys Mechanical APDL.
```

File: examples/03-advanced/05-extrapolation_strain_2d.py [hunk 35]
```diff
@@ -3,16 +3,25 @@
...
 
-4th step: compare the results between nodal elastic strain from data source ref and nodal strain computed by Extrapolation Method.
+4th step: compare the results between nodal elastic strain from data source
+ref and nodal strain computed by Extrapolation Method.
```
Comment (akaszynski) [inline examples/03-advanced/05-extrapolation_strain_2d.py:24]:
```suggestion
4th step: Compare the results between nodal elastic strain from the data
source and nodal strain computed by extrapolation method.
```

File: .flake8 (hunk 1 above)
Comment (jleonatti) [inline .flake8:2]:
```suggestion
```

File: examples/03-advanced/05-extrapolation_strain_2d.py [hunk 36]
```diff
@@ -39,7 +48,8 @@
 ###############################################################################
 # Extrapolation from integration points for elastic strain result
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-# In this example we compute nodal component elastic strain results from integration points's elastic strain by using operator gauss_to_node_fc
+# In this example we compute nodal component elastic strain results from
+# integration points's elastic strain by using operator gauss_to_node_fc
```
Comment (akaszynski) [inline examples/03-advanced/05-extrapolation_strain_2d.py:52]:
```suggestion
# the elastic strain at the integration points  by using the ``gauss_to_node_fc``
# operator.
```

File: examples/03-advanced/05-extrapolation_strain_2d.py [hunk 37]
```diff
@@ -49,8 +59,13 @@
 ###############################################################################
 # Nodal elastic strain result of integration points:
 ###############################################################################
-# The command "ERESX,NO" in Mechanical APDL is used to copy directly the gaussian (integration) points results to the nodes, instead of the results at nodes or elements (which are interpolation of results at a few gauss points).
-# The following plot shows the nodal values which are the averaged values of elastic strain at each node. The value shown at the node is the average of the elastic strains from the gaussian points of each element that it belongs to.
+# The command "ERESX,NO" in Mechanical APDL is used to copy directly the
```
Comment (akaszynski) [inline examples/03-advanced/05-extrapolation_strain_2d.py:62]:
```suggestion
# The command ``ERESX,NO`` in Mechanical APDL is used to copy directly the
```

File: examples/03-advanced/05-extrapolation_strain_2d.py [hunk 38]
```diff
@@ -59,7 +74,8 @@
 
 
 ###############################################################################
-# Create operator gauss_to_node_fc and compute nodal component elastic strain by applying Extrapolation Method
+# Create operator gauss_to_node_fc and compute nodal component elastic strain
```
Comment (akaszynski) [inline examples/03-advanced/05-extrapolation_strain_2d.py:77]:
```suggestion
# Create ``gauss_to_node_fc`` operator and compute nodal component elastic strain
```

File: examples/03-advanced/05-extrapolation_strain_2d.py [hunk 39]
```diff
@@ -59,7 +74,8 @@
...
+# by applying Extrapolation Method
```
Comment (akaszynski) [inline examples/03-advanced/05-extrapolation_strain_2d.py:78]:
```suggestion
# Create the ``gauss_to_node_fc`` operator and compute nodal component 
# elastic strain by applying the extrapolation method.
```

File: examples/03-advanced/05-extrapolation_strain_2d.py [hunk 40]
```diff
@@ -98,7 +114,8 @@
 # Compare the elastic strain result computed by extrapolation and reference's result.
 # Check if two fields container are identical.
 # Maximum tolerance gap between to compared values: 1e-3.
-# Smallest value which will be considered during the comparison step : all the abs(values) in field less than 1e-14 is considered as null
+# Smallest value which will be considered during the comparison
+# step : all the abs(values) in field less than 1e-14 is considered as null
```
Comment (akaszynski) [inline examples/03-advanced/05-extrapolation_strain_2d.py:118]:
```suggestion
# step : all the ``abs(values)`` in the field less than 1e-14 are considered null
```

File: examples/04-specific-requests/00-hdf5_double_float_comparison.py [hunk 41]
```diff
@@ -68,6 +68,9 @@
 
 ###############################################################################
 # Comparison
+float_precision = os.stat(os.path.join(tmpdir, "c:/temp/dpf_float.h5")).st_size
+double_precision = os.stat(os.path.join(tmpdir, "c:/temp/dpf_double.h5")).st_size
```
Comment (akaszynski) [inline examples/04-specific-requests/00-hdf5_double_float_comparison.py:72]: Way more readable.  Thanks!

File: test.py [hunk 42]
```diff
@@ -0,0 +1,10 @@
+from ansys.dpf import core as dpf
+from ansys.dpf.core import examples
+
+
+class A:
+    def b():
+        my_name_is_model = dpf.Model(examples.msup_transient)
+        my_name_is_model.results.displacement.on_last_time_freq.on_named_selection(
+            "_CONSTRAINEDNODES"
+        )
```
Comment (akaszynski) [inline test.py:10]: Curious, why was this added?

**Review by akaszynski — APPROVED** [review]
Approved with various suggestions.

File: test.py (hunk 42 above)
Comment (jleonatti) [reply test.py:10]: This was a test module I made to properly format a docsring example that was too long. It should not have been committed. I will remove it!

File: ansys/dpf/core/time_freq_support.py [hunk 43]
```diff
@@ -430,12 +430,16 @@ def append_step(
...
+        >>> tfq2.append_step(
+        ...    1, [0.1, 0.21, 1.0], rpm_value = 2.0, step_harmonic_indices = [1.0, 2.0, 3.0]
+        ... )
         >>> tfq2.append_step(2, [1.1, 2.0], rpm_value = 2.3, step_harmonic_indices = [1.0, 2.0])
         >>> tfq2.append_step(3, [0.23, 0.25], rpm_value = 3.0, step_harmonic_indices = [1.0, 2.0])
 
         >>> tfq3 = TimeFreqSupport()
-        >>> tfq3.append_step(1, [0.1, 0.21, 1.0], rpm_value = 2.0, step_harmonic_indices = { 1 : [1.0, 2.0, 3.0], 2 : [1.0, 2.0, 2.5] } )
+        >>> tfq3.append_step(1, [0.1, 0.21, 1.0], rpm_value = 2.0,
+        ...     step_harmonic_indices = { 1 : [1.0, 2.0, 3.0], 2 : [1.0, 2.0, 2.5] }
```
Comment (akaszynski) [inline ansys/dpf/core/time_freq_support.py:441]:
```suggestion
        ...     step_harmonic_indices = {1: [1.0, 2.0, 3.0], 2: [1.0, 2.0, 2.5]}
```

**Comment by akaszynski** [comment]
Doctest still needs to be fixed to use `pytest -v --doctest-modules pyvista`, otherwise you have to fix these one by one.

**Review by rlagha — APPROVED** [review]
